src/components/PopOver/index.js

```diff
@@ -69,7 +71,7 @@ class PopOver extends Component {
    * Viewport size
    * @reduce(object) - additional top/bottom screen reduction cause by sticky header/footer
    */
-  static calculatePosition(position, dimensions, reduce) {
+  static calculatePosition(position, dimensions, reduce, inlineWithTarget) {
```

Comment:
I'd suggest wrapping args into a single object: `static calculatePosition({ position, dimensions, reduce, inlineWithTarget })` since when there are more than 2  args, it becomes hard to remember their order.

Reply:
Updated 🙂